### PR TITLE
Increase timeout to 5 mins

### PR DESCRIPTION
**Pull request by nnegrey**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/infiniteStreaming.js [hunk 1]
```diff
@@ -271,7 +271,7 @@ require(`yargs`)
     },
     streamingLimit: {
       alias: 's',
-      default: 10000,
+      default: 290000,
```
Comment (stephenplusplus) [inline samples/infiniteStreaming.js]: Just wondering, why not 300000 for 5 minutes even?

Comment (leahecole) [reply samples/infiniteStreaming.js]: +1 was wondering the same thing

Comment (nnegrey) [reply samples/infiniteStreaming.js]: If we wait for the full 5 minutes, then the API will return an error that we've reached the limit, so we cut it off a few seconds early and then restart the stream with the buffered audio to provide a sort of "infinite" streaming.

Comment (stephenplusplus) [reply samples/infiniteStreaming.js]: That would have been a cool comment to leave in the code :)

**Pull request merged**